packages/techdocs-common/src/stages/prepare/types.ts

```diff
@@ -38,15 +91,20 @@ export type PreparerBase = {
    *        updated since the last build.
    * @throws `NotModifiedError` when the prepared directory has not been changed since the last build.
    */
-  prepare(
-    entity: Entity,
-    options?: { logger?: Logger; etag?: string },
-  ): Promise<PreparerResponse>;
+  prepare(entity: Entity, options?: PreparerOptions): Promise<PreparerResponse>;
 };
 
+/**
+ * Definition for a Tech Docs preparer builder
```

Comment:
_Vveeeerrry_ minor: recommend consistency on `TechDocs` vs. `Tech Docs` (throughout this PR)

Reply:
Fixed!